Makefile

```diff
@@ -43,11 +44,16 @@ endif
 KUBE_VERSION ?= 1.21
 KIND_CONFIG ?= kind-$(KUBE_VERSION).yaml
 
+ensure-generate-is-noop: generate bundle
+	cat ./bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml
```

Comment:
I'd keep this out. If anything, we can just show the diff. Otherwise, it will spit out this file every time, including when we don't need it.

Reply:
This is just to show that the issue. I will definitely undo this line before merging.... Just to confirm it in CI I added this for now. :)